### DISCUSSION TITLE
Tildes and ñ in the text

### DISCUSSION THREAD

**Discussion by alfredorodrigalvarez** [ruffle-rs/ruffle#4363] [General]
With static text, accents appear.
With dynamic text, accents do not appear.

![AcentosSINO](https://user-images.githubusercontent.com/84404451/118696777-ecfd2c80-b80e-11eb-9a40-07822e5164cf.png)

¿Podrían corregirlo?

Gracias

**Comment by ousia** [comment]
Hola @alfredorodrigalvarez,

como no incluyes el archivo SWF, conjeturo sobre qué pasa con él. No creo que me equivoque.

Aunque podrías añadirlo directamente, arrastrando sobre  la ventana del comentario. Como no se admiten directamente archivos SWF, debes añadirle la extension `.txt` para que lo admita.

Me temo que el problema es un poco más complicado que lo que dices. No tiene que ver con que el campo de texto sea dinámico.

Está relacionado con el hecho de que el campo de texto use un tipo de letra que no incrustado en el archivo SWF.

Debido a que de momento _Ruffle_ no puede usar las tipografías de sistema operativo (#1862), el rodeo temporal es que se use un tipo de letra incrustado en _Ruffle_.

Sólo funciona con inglés, porque el tipo incrustado no tiene caracteres acentuados (con diacríticos, lo que incluye _ñ_).

Si estás muy interesado en colaborar en que _Ruffle_ pueda solucionar esto, en https://github.com/ruffle-rs/ruffle/issues/1862#issue-757869643 está lo que se necesita para hacerlo.

Bienvenido al proyecto. Sólo soy un usuario.

**Comment by alfredorodrigalvarez** [comment]
Buenas tardes: Tenía varias aplicaciones que hice sobre lectura para mis
alumnos y observé que en unas salían las tildes y en otra no. Las
había hecho con flash 8.0. Las comprobé y vi que en los campos de texto que
eran dinámicos no salían las tildes. Los corregí poniéndolos en texto
estático como estaban en los que sí se leían las tildes y... ¡Voila! ya
aparecían las tildes con Ruffle y el tipo de letra en todas las
aplicaciones era el mismo, creo que era Arial. por ello creo que aquí está
la solución; o por lo menos una de ellas.

Un saludo.

**Reply by ousia** [reply]
@alfredorodrigalvarez,

como regla general (e indepedientemente de cómo funcione _Ruffle_), es buena idea incrustar las tipografías en los documentos que necesitan usarlas.

Desde luego, es claro en documentos PDF. Creo que no es mala idea hacerlo también en archivos SWF.

Por cierto, si la pregunta está contestada, es bueno marcarla como tal. No lo digo por mí, sino porque las respuestas contestadas se muestren como tales. A diferencia de las incidencias (que se cierran), las respuestas han de marcarse como contestadas por quien pregunta.

Me alegro de que hayas encontrado un rodeo que evita el problema.
